src/NUnitFramework/tests/Api/TestAssemblyRunnerTests.cs

```diff
@@ -433,71 +445,76 @@ public void RunAsync_BadFile_ReturnsNonRunnableSuite()
                 Does.StartWith("Could not load"));
         }
 
-#endregion
+        #endregion
```

Comment:
This doesn't line up with other region directives in the file anymore.

Reply:
I stopped cleaning these up when VS shifts them, because I understood they were all being removed at some point. Is that not the case?

Reply:
@ChrisMaddock offered to remove them all from the https://github.com/nunit/nunit-console repo in https://github.com/nunit/nunit-console/issues/635. If you like I can open an issue and probably mark it `good first issue` to do this same in this codebase.

Reply:
I wouldn't like to have a global edit issue out there as a `good first issue` unless we 100% agree that every single `#region` directive should be removed. As I expressed elsewhere, I'd rather not see global changes at all. And that's __even__ (in case it wasn't clear) if I agree with the change that is being made. I think global changes for conformity or consistency are evil!. :smiling_imp: 

OTOH if we had a standard that said NEVER use `#region` or even AVOID it, I'd be removing them in every file I touched, just like I add `_` in front of names. I think that's a better way to establish a team coding standard than doing blanket changes because it gets everyone's hands dirty, so to speak.

Reply:
If we're discussing personal preferences, far from being evil, I think consistency is lovely because it puts the focus on the logic itself.

> I stopped cleaning these up when VS shifts them, because I understood they were all being removed at some point. Is that not the case?

If you don't want us to remove them as a global edit, and you don't want to remove them as you touch these files, would you mind making the indentation consistent where you are touching them?

Reply:
Sorry, I thought we were done and merged. 

To be clear, I __would__ want to remove them in this PR if that were the team standard. I'd want to have the wording first... is it NEVER or AVOID?

I do admit to not wanting to spend time on something the IDE keeps changing on me.

Reply:
Self-consistency is great. Imposed consistency (by or on others) not so great. Consistency that arises out of a team is best of all but hardest to achieve.

Reply:
@rprouse Do you want to make region removal a team standard so that Charlie feels comfortable removing them in this PR? Or would you want to do the same thing Chris did in nunit-console?

Reply:
@jnm2 That would be "imposed" of course. That is, whether Rob sets it as a team standard OR decides to make the change himself as Chris did, it would still be happening without a discussion. So be it.

But even so it still requires a clear statement of what the standard actually is. I'm using the MS terminology above: ALWAYS, USUALLY, AVOID, NEVER, which I think is a nice way to clarify a standard. It can be done in other ways, of course, but putting it into words is often a way to find out how clear your thinking is on the issue.

Sorry for my confusion about this PR, which I thought we had merged already. It was the related GUI PR that I merged. Obviously, this one is unmerged,  even though approved. Since you're concerned about the `#region` issue, I'll wait to merge. Gotta say though, I think that being able to cancel tests ought to have priority over an issue like this one.

Reply:
I'm finding this very complex and trying to do the complexity justice, but there are foure paths:

- Squashing out the region indentation to fix the inadvertent changes. I avoid asking for this because it's a bunch of additional steps.
- Removing the regions in this PR that were touched. You said you'd do this if we had a standard with the terminology AVOID or NEVER, which means this option is now blocked unless we wait for a team discussion.
- Leaving the autoindentation thrashing added in this PR, and opening an issue so we remember to remove all regions. You said you thought we were already going to remove all regions, and then that you would not like me to open an issue to track this and that you're against global changes.
- Leaving the autoindentation thrashing added in this PR and forgetting about it. 

Of *course* being able to cancel tests ought to have priority over formatting. By attaching complexity, which I don't agree with but which we're both loathe to turn into a metadiscussion, you have systematically left no path open other than disregarding my comment. This is fine; I wouldn't have approved if I thought it was important. I just wish you didn't make it sound like I'm suggesting that we prioritizing style discussions over real bug fixes.

Reply:
Well Sh*t, I typed a comment and then unresolved the conversation and my comment disappeared. I __hate__ this IDE. I'll start over when I calm down!

Reply:
So a couple of points...

 * Editing the `#region` statements that were changed by VS. You seem to have misunderstood me. I said I __couldn't__ do it because the PR was merged. I was under the impression you had merged it and I wasn't looking at it's status. I actually mentioned this in two comments. I was about to remind you of it again when I noticed that the PR had __not__ been merged. So I made the change you asked for as I would naturally do for any reasonable change request.

I __thought__ a global decision had been made to remove `#region` statements. You corrected me and said it was only made for nunit-console... at least that's what I think you said. But then you asked me to make the change in the files I had touched anyway. That was a really strange request, particularly since it's a change I would argue against in any discussion we might have.

That said, my views about `#region` are pretty flexible compared to my views about how decisions ought to get made and documented.

Finally, it's not __I__ who "makes it sound like" you prioritize style discussions over bug fixes. I'm just echoing back what I get from what your comments focus on and from your having referred the PR to @rprouse  to resolve.

Reply:
It sounds like at least some of the conversation was talking past each other. We have some very different sets of expectations about what things mean. As far as my comments focusing on style, I only continued replying because you seemed to be suggesting options or requirements that needed a response. It was frustrating to hear that from your point of view I was putting style above getting this PR through. If you were saying I should just drop it, I apologize for missing that.

I did not refer this PR for Rob to resolve; I asked him question about our future plans to see which course to take out of the options you were presenting. This is again frustrating; you're attributing things to me that I was not aware of. Why must our conversations be this way?

Reply:
Let me swallow my frustration and put this on myself, since as far as I know that's where the problem is happening. It's probably fair to say this would be compatible with your perspective on the conversation. 😊

Do you have any tips—besides trying to be underfoot less—for how I can better listen or better communicate my thoughts? If I don't figure something out that I can act on to improve my communication, I'm afraid I face two unpleasant options: engage and frustrate you, or withdraw at a hint of disagreement which seems almost passive aggressive and is in any case naturally hard for my personality. Both options seem like things you wouldn't want either.

Reply:
I didn't think I was presenting options or making things more complex. I just saw myself as giving you reasons why I should not remove the #region statements, something which I both disagree with on it's merits and don't think should be part of this PR in any case. I thought that was pretty clear.

My suggestion is not to bring things up in a PR unless they impact the PR's approval. If you pull the project lead in, that's presumably to resolve a disagreement between us about the PR... because we are in a code review of the PR, not a general discussion! When you're the reviewer, you're the authority, and the project lead is an even bigger authority.

Frankly, at this point, I'm shocked that the PR has been merged without any discussion of the actual code changes. I'd like to think it was so good that nobody could find anything to change, but that's unlikely. I feel like all the attention went to the #region stuff and not to the meat of the actual change.

Reply:
My comment asking whether we could move the event subscription out of the lock doesn't count? All my attention did not go to the regions. They were an afterthought. There was nothing fundamental I saw that should change.

Reply:
I'll keep those things in mind for your next PR, and I'll clarify to counter the presumption if I do want to tag Rob (I don't know the reason I've tagged him was ever in order to arbitrate a disagreement). Thanks for your advice!

My comment asking whether we could move the event subscription out of the lock doesn't count? All my attention did not go to the regions. They were an afterthought. There was nothing fundamental I saw that should change.